test/harry/main/org/apache/cassandra/harry/cql/SelectHelper.java

```diff
@@ -146,7 +145,7 @@ public static CompiledStatement select(Operations.SelectCustom select, SchemaSpe
             }
         }
 
-        builder.allowFiltering();
+        builder.withAllowfiltering();
```

Comment:
`allowFiltering` was the only method in the class that wasn't prefixed with `with`, so migrated to match the style of the class